.github/workflows/osrm-backend-docker.yml

```diff
@@ -0,0 +1,30 @@
+name: build and publish container image
+on:
+  create:
+    tags:
+      - v*
```

Comment:
I think tag filters are only supported on the `push` or `pull_request` events
https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#onpushpull_requestbranchestags

For example, it looks like this triggered when you created a branch with no tag.
https://github.com/hoerup/osrm-backend/actions/runs/1279643123